packages/core/src/compiler/Style.ts

```diff
@@ -117,7 +117,7 @@ import {
 import { checkTypeConstructor, isDeclaredSubtype } from "./Domain";
 
 const log = consola
-  .create({ level: LogLevel.Info })
+  .create({ level: LogLevel.Warn })
```

Comment:
Is this change relevant to this PR, or just for general quality-of-life?

Reply:
The latter, mostly annoyed by the console haha